/**:
  ros__parameters:
    color_range_list:
      black:
        max:
        - 86
        - 187
        - 196
        min:
        - 0
        - 30
        - 92
      blue:
        max:
        - 172
        - 178
        - 119
        min:
        - 11
        - 105
        - 59
      green:
        max:
        - 203
        - 119
        - 164
        min:
        - 46
        - 55
        - 130
      red:
        max:
        - 185
        - 183
        - 189
        min:
        - 9
        - 142
        - 130
      tennis:
        max:
        - 210
        - 159
        - 255
        min:
        - 31
        - 66
        - 178
      white:
        max:
        - 255
        - 155
        - 155
        min:
        - 111
        - 100
        - 100
      yellow:
        max:
        - 255
        - 177
        - 184
        min:
        - 195
        - 110
        - 146
